### PR TITLE
Replace events with generic event 'run'

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by helfi92** [comment]
Makes sense, sorry I misread.

File: packages/neutrino/src/index.js [hunk 1]
```diff
@@ -15,12 +16,18 @@ const run = (command, middleware, options) => {
     .map(() => api.config.merge(options.config))
     // Trigger all pre-events for the current command
     .chain(() => Future.fromPromise2(api.emitForAll, `pre${command}`, api.options.args))
+    // Trigger generic pre-event
+    .chain(() => Future.fromPromise2(api.emitForAll, `pre${genericEvent}`, api.options.args))
```
Comment (eliperelman) [inline packages/neutrino/src/index.js]: I'm fine with inlining the `prerun` name here:

```js
.chain(() => Future.fromPromise2(api.emitForAll, `prerun`, api.options.args))
```

File: packages/neutrino/src/index.js [hunk 2]
```diff
@@ -15,12 +16,18 @@ const run = (command, middleware, options) => {
...
     // Execute the command
     .chain(() => api.run(command))
     // Trigger all post-command events, resolving with the value of the command execution
     .chain(value => Future
       .fromPromise2(api.emitForAll, command, api.options.args)
       .chain(() => Future.of(value)))
+    // Trigger generic post-event, resolving with the value of the command execution
+    .chain(value => Future
+      .fromPromise2(api.emitForAll, genericEvent, api.options.args)
```
Comment (eliperelman) [inline packages/neutrino/src/index.js]: I'm fine with inlining the `run` name here:

```js
.fromPromise2(api.emitForAll, 'run', api.options.args)
```

**Comment by eliperelman** [comment]
🎉

**Pull request merged**